rpc/client/examples_test.go

```diff
@@ -20,7 +21,7 @@ func ExampleHTTP_simple() {
 	rpcAddr := rpctest.GetConfig().RPC.ListenAddress
 	c, err := client.NewHTTP(rpcAddr, "/websocket")
 	if err != nil {
-		panic(err)
+		log.Fatal(err)
```

Comment:
are these functions test helpers? examples? something else?

Reply:
https://golang.org/pkg/log/#Fatal

stdlib functions. I prefer to use them in example tests instead of panics.

Reply:
Sorry, I meant the functions in this file. I know we're trying to "reduce panics" but since log.Fatal functions just like a panic, I'm wondering if there's a better alternative. 

For example, if these functions are test helpers, we could pass a `t.Testing` in and then call `t.Fatal`.

Reply:
Normally, we'd use t.Testing, but since these are example tests https://blog.golang.org/examples, log.Fatal is preferable (that's what stdlib is using https://golang.org/src/net/http/httptest/example_test.go)

Reply:
Cool, that answers my original question--I wasn't sure if they were examples, test helpers, or actual tests.